tests/chunks.test.js

```diff
@@ -9,7 +9,7 @@ const sqldb = require('@prairielearn/postgres');
 const courseDB = require('../sync/course-db');
 const chunksLib = require('../lib/chunks');
 const config = require('../lib/config');
-const logger = require('./dummyLogger');
+const { makeMockLogger } = require('./mockLogger');
```

Comment:
These changes and those in `helper*` were made in the interest of not duplicating our "sync the test course" logic again.